content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/clientlibs/editor/js/tabs.js

```diff
@@ -32,17 +32,19 @@
             var activeSelect = tabsEditor.querySelector(activeSelectSelector);
             var activeItem = tabsEditor.querySelector(activeItemSelector);
 
-            Coral.commons.ready(childrenEditor, function() {
-                updateActiveSelect(childrenEditor, activeSelect, activeItem);
-            });
+            if (childrenEditor && activeSelect && activeItem) {
+                Coral.commons.ready(childrenEditor, function() {
+                    updateActiveSelect(childrenEditor, activeSelect, activeItem);
```

Comment:
I would prefer to encapsulate the checks within the `updateActiveSelect` function.

Reply:
We should still check for each element here, as we bind events to them:
```
activeSelect.on("change", function() {
                    activeItem.value = activeSelect.value;
});
```

Reply:
to have a more robust function I would still add the checks in the function.

Reply:
Updated in 4838abb6ddcd2345bcc307daebfca42d9b9fafb4